### ISSUE TITLE
New Pattern for unnecessary continue

### ISSUE THREAD

**Issue by andreasschroth** [kalessil/phpinspectionsea#1179]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | New Pattern                          |
| Plugin         | Php Inspections (EA Ultimate), 2.0.12 |
| Language level | PHP 7.3                        |

Current behaviour:
---

Here is some example code:

```php
<?php

declare(strict_types=1);

$foo = [];

foreach ($foo as $bar) {
    try {
        echo 'Yep!';
    } catch (Exception $exception) {
        continue;
    }
}
```

No warning is displayed.

Suggested behaviour:
---

After the catch, no more code is executed, thus the `continue;` is unnecessary here.

**Comment by kalessil** [comment]
That\s a neat pattern, thank you @andreasschroth !

**Comment by kalessil** [comment]
Implemented!
